src/commands/connections/updateWebAppSettings.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AppSettingsTreeItem } from "vscode-azureappservice";
+import { ext } from "../../extensionVariables";
+
+export async function updateWebAppSetting(webAppId: string, appSettingToUpdate: string, value: string): Promise<void> {
+    const appSettItem = <AppSettingsTreeItem | undefined>await ext.tree.findTreeItem(webAppId + String('/application'));
```

Comment:
I don't like using `findTreeItem` with `/application` hard-coded. Instead, you should use the node's parent to eventually get the app settings tree item. The default type for a parent is `AzureParentTreeItem | undefined`, but you can make that type more specific like this:
```
export class CosmosDBDatabase extends AzureTreeItem<ISiteTreeRoot> {
    ...
    public readonly parent: CosmosDBTreeItem;

    constructor(parent: CosmosDBTreeItem, ...) {
        super(parent);
        ...
    }
}
```

Once you get to the WebAppTreeItem parent, you can access `appSettingsNode` directly: https://github.com/Microsoft/vscode-azureappservice/blob/master/src/explorer/WebAppTreeItem.ts#L26